doc/src/challenges/0014-intrinsics-simd.md

```diff
@@ -0,0 +1,163 @@
+# Challenge 14: High-Assurance SIMD Intrinsics for Rust
```

Comment:
Random place for comment: please add a link to `doc/src/SUMMARY.md`.

Reply:
Done.